.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           if [[ ${{matrix.os}} == macos* ]]; then
             /bin/bash -c "$(curl -fsSL \
               https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-            brew install automake fop wxwidgets fish shellcheck shfmt
+            brew install automake fop wxwidgets fish shellcheck shfmt coreutils
```

Comment:
Otherwise `nproc` is not available on macOS, which might make for slower CI build times.